portal-bridge/src/types/e2s.rs

```diff
@@ -108,16 +124,16 @@ impl Entry {
     }
 }
 
-#[derive(Decode, Encode)]
-struct Header {
-    entry_type: u16,
-    length: u32,
-    reserved: u16,
+#[derive(Clone, Debug, Decode, Encode, Default, Eq, PartialEq)]
+pub struct Header {
+    pub type_: u16,
```

Comment:
There there a reason for the _ here? `` pub type_: u16,``

Reply:
`type` is a reserved keyword in rust, so the trailing `_` is required to distinguish it from `type`